include/net/cloud.h

```diff
@@ -149,13 +135,11 @@ struct cloud_backend {
  * @param backend Pointer to cloud backend structure.
  * @param handler Handler to receive events from the backend.
  */
-static inline int cloud_init(
-	struct cloud_backend *const backend,
-	cloud_evt_handler_t handler)
+static inline int cloud_init(struct cloud_backend *const backend,
+			     cloud_evt_handler_t handler)
```

Comment:
Alignment issue? Looks like clang-format only does tab-indentation, and doesn't fill in the remaining spaces?